pkgs/applications/audio/spotify/default.nix

```diff
@@ -9,7 +9,7 @@ let
   # Latest version number can be found at:
   # http://repository-origin.spotify.com/pool/non-free/s/spotify-client/
   # Be careful not to pick the testing version.
-  version = "1.0.70.399.g5ffabd56-26";
+  version = "1.0.72.117.g6bd7cc73-35";
```

Comment:
Is this the latest stable or is this a testing version?

Reply:
probably it's the testing version, the older one is always the stable one, is it?